### PR TITLE
DOC: update code style for development doc and user guide #36777

### PR DESCRIPTION
**Pull request by karasinski**
Addresses part of #36777
Ran blacken-tools and checked for warnings from flake8-rst for 4 additional doc files.

### CODE REVIEW

File: doc/source/user_guide/dsintro.rst [hunk 1]
```diff
@@ -556,17 +564,16 @@ derived from existing columns.
 
 .. ipython:: python
 
-   iris = pd.read_csv('data/iris.data')
+   iris = pd.read_csv("data/iris.data")
    iris.head()
-   (iris.assign(sepal_ratio=iris['SepalWidth'] / iris['SepalLength'])
-        .head())
+   (iris.assign(sepal_ratio=iris["SepalWidth"] / iris["SepalLength"]).head())
```
Comment (dsaxton) [inline doc/source/user_guide/dsintro.rst:569]:
```suggestion
   iris.assign(sepal_ratio=iris["SepalWidth"] / iris["SepalLength"]).head()
```

File: doc/source/user_guide/dsintro.rst [hunk 2]
```diff
@@ -663,6 +672,31 @@ row-wise. For example:
 
    df - df.iloc[0]
 
+In the special case of working with time series data, if the DataFrame index
```
Comment (dsaxton) [inline doc/source/user_guide/dsintro.rst:675]: This is being added to the documentation?

File: doc/source/user_guide/visualization.rst [hunk 3]
```diff
@@ -117,10 +116,10 @@ For example, a bar plot can be created the following way:
 
 .. ipython:: python
 
-   plt.figure();
```
Comment (dsaxton) [inline doc/source/user_guide/visualization.rst:120]: I think these semicolons may have been added intentionally to suppress certain output, I'd leave them for the purposes of this

Comment (karasinski) [reply doc/source/user_guide/visualization.rst:120]: Okay, I returned all the semicolons to the `plt.figure()` lines that had them.

File: doc/source/user_guide/dsintro.rst (hunk 2 above)
Comment (karasinski) [reply doc/source/user_guide/dsintro.rst]: Removed--I was attempt to resolve a merge error (and inadvertently added this).

**Comment by karasinski** [comment]
I believe I've responded to your comments, let me know if there are any additional changes to be made.

**Review by dsaxton — APPROVED** [review]

**Comment by dsaxton** [comment]
Thanks @karasinski

**Pull request merged**